contributed_definitions/nyaml/NXcs_mm_obj.yaml

```diff
@@ -5,7 +5,7 @@ symbols:
   doc: |
     The symbols used in the schema to specify e.g. dimensions of arrays.
 type: group
-NXcs_mm_obj(NXobject):
+NXcs_mm_obj(NXcircuit):
```

Comment:
see comment on `NXcs_io_obj`

Reply:
Agree, followed your suggestion and refactored this set of base classes, only NXcs_computer remains using NXcircuit.